packages/app/universal-ts-utils/src/public/api-contracts/apiContracts.ts

```diff
@@ -135,13 +135,13 @@ export function buildPayloadRoute<
 }
 
 export function buildGetRoute<
-  PathParams,
   ResponseBodySchema extends z.Schema | undefined = undefined,
-  PathParamsSchema extends z.Schema<PathParams> | undefined = undefined,
+  PathParamsSchema extends z.Schema | undefined = undefined,
```

Comment:
@CarlosGamero Extracted to https://github.com/lokalise/shared-ts-libs/pull/471 and extended to all methods